src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -15,12 +16,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session as Session);
     const update = appData.update as Update;
 
+    let journeyQuery = "";
+    let journeyType = config.JourneyType.update;
+    if (isRemoveJourney(req)) {
+      journeyType = config.JourneyType.remove;
+      journeyQuery = "?joureny=remove";
```

Comment:
typo in the query , but it is also available in the config, it's the same one you've used on line 27

Reply:
Removed this and rewritten it elsewhere.